### ISSUE TITLE
Harmonize Logging to use Log4j2 or at least one common logging

### ISSUE THREAD

**Issue by blaumeiser-at-bosch** [eclipse/antenna#385]
### Summary of the Bug

Currently we have a configuration for log4j and log4j2, Ort seems to use another logging technology which does some logging in between, ...

This should be harmonized to have only one logging used throughout the tooling.

In addition there is a log message from the ORT Result analyzer which seems to come from the used ORT libraries, this information should not be logged.

### Steps to reproduce
- Observe core/runtime/src/main/resources/log4j*.properties
- Observe when runnig the antenna cli that there is a ort log message

### Acceptance Criteria
- [x] One configuration is setup with one common backend
- [x] No special ORT log messages, they should be configured out

### Definition of Done
- Acceptance criteria fulfilled
- A test case is created to reproduce the bug
- A PR is created, the CI infrastructure reports green, the bug test case proves that bug is fixed
- The PR is reviewed and approved
- No TODOs left in the code unless explained in the ticket, if something else is still open, this is summarized in a comment in the issue
- Documentation is updated

**Comment by sschuberth** [comment]
> In addition there is a log message from the ORT Result analyzer which seems to come from the used ORT libraries, this information should not be logged.

Actually, why not? If the ORT analyzer is used, you might want to know if something goes wrong, and log output can be really helpful then. I'd rather configure the logger in a way that the FQN of the class is being logged, so that it becomes evident whether the log output originates from ORT, or Antenna directly.

**Comment by blaumeiser-at-bosch** [comment]
> > In addition there is a log message from the ORT Result analyzer which seems to come from the used ORT libraries, this information should not be logged.
> 
> Actually, why not? If the ORT analyzer is used, you might want to know if something goes wrong, and log output can be really helpful then. I'd rather configure the logger in a way that the FQN of the class is being logged, so that it becomes evident whether the log output originates from ORT, or Antenna directly.

I reduced the normal logging to a minimum, so that if you run antenna, you get basically progress information on which step is currently active, if no warn or error is created each step logs not much except for long lasting activities. In this output sequence the ORT log information looks somewhat detailed compared to the rest, so pure aesthetics.

**Comment by oheger-bosch** [comment]
I did some experiments and tweaked the logging configuration locally. All logging output is now processed by log4j 2, and the configuration file for log4j 1 is obsolete. (Actually, I assume that this setup has already been active before because of the logging libraries pulled in by transitive dependencies. I made the dependencies more explicit and added additional adapters to make sure that logging output generated via other logging frameworks is always redirected to log4j 2.)

When antenna runs integrated with a build tool (as maven or gradle plugin) a very basic logging configuration can be used that just prints the logging message. It is then up to the build tool to handle formatting and filtering; this makes sure that the output from antenna looks the same as the other logging information generated by the tool.

For the CLI we have more options. For instance, I tested with a pattern layout that displays the time of the log event, the log level, and the class that generated the output. I also set the overall logging level to warn and enabled info logging only for the antenna packages. This has the effect that the verbose logging from ORT is suppressed. But just tell me your preferences.
